core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/dot-edit-content.component.html

```diff
@@ -66,11 +66,14 @@
 </div>
 <div
     class="dot-edit-content__palette"
-    *ngIf="isEnterpriseLicense && isEditMode"
+    *ngIf="isEnterpriseLicense && isEditMode && allowedContent"
```

Comment:
I added that conditional because when the page loads the variable `allowedContent` is `undefined`. It takes another request to determine the `allowedContent`. That's why I don't want to init the `dot-palette` without known the `allowedContent`. In case that there is not `allowedContent` for the `page`, I set the variable an empty array to pass the condition and the `dot-palette` is going to show only `widgets`.

https://github.com/dotCMS/core/blob/1571de78379b1df8b2c747b8e55e4f8d416e0d9a/core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts#L261-L269

Reply:
We can also move this logic to the `dot-pallet` component so we won't need the condition.